cookbook/doctrine/registration_form.rst

```diff
@@ -76,9 +76,9 @@ You have a simple ``User`` entity mapped to the database::
     }
 
 This ``User`` entity contains three fields and two of them (``email`` and
-``plainPassword``) should display on the form. The email property must be unique
-in the database, this is enforced by adding this validation at the top of
-the class.
+``plainPassword``) should be displayed by the form. The email property must be
+unique in the database, this is enforced by adding an ``@UniqueEntity``
```

Comment:
``@UniqueEntity`` is not enforcing it in the DB. It is enforcing it in the Symfony validation. If you want a unique constraint in the DB as well (and you should), you need to add ``unique=true`` in the column mapping